lib/publiccloud/instance.pm

```diff
@@ -64,7 +64,7 @@ sub run_ssh_command {
         $cmd = "'$cmd'";
     }
 
-    my $ssh_cmd = sprintf('ssh %s "%s@%s" -- %s', $args{ssh_opts}, $args{username}, $self->public_ip, $cmd);
+    my $ssh_cmd = sprintf('ssh %s "%s@%s" -- %s &> /dev/%s', $args{ssh_opts}, $args{username}, $self->public_ip, $cmd, $serialdev);
```

Comment:
I guess it will hide the output for `script_output()` or?

Reply:
True, my bad. I will change it.

Reply:
Can you look now Clemens?